pkgs/tools/audio/beets/common.nix

```
@@ -137,8 +146,26 @@ python3Packages.buildPythonApplication rec {
     runHook postCheck
   '';
 
+
   passthru.plugins = allPlugins;
 
+  passthru.tests.beets = runCommand "beets-tests" {
```

Let's be more specific:

```suggestion
  passthru.tests.gstreamer = runCommand "beets-gstreamer-test" {
```